### PR TITLE
chore: generate initial project structure

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- chore: generate initial project structure

Deploy Link: https://shorebirdtech.github.io/docs

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [X] 🗑️ Chore

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (felangel) [inline docs/index.md:1]: This is where we'll write our docs using markdown (mdx is also supported 🎉 ).

File: src/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+/* You can override the default Infima variables here. */
+/* You can override the default Infima variables here. */
+:root {
+  --ifm-color-primary: #2a48df;
```
Comment (felangel) [inline src/css/custom.css:10]: We need to decide on a color scheme, I just kept the defaults for now.

File: static/img/logo.svg
Comment (felangel) [inline static/img/logo.svg:1]: @eseidel should I use the logo from `shorebird.dev` for now?

Comment (eseidel) [reply static/img/logo.svg:1]: Sure.

File: src/css/custom.css (hunk 2 above)
Comment (eseidel) [reply src/css/custom.css:10]: We have a color scheme in index.html with a link to where I generated it.  That's probably the thing to start with.

**Review by eseidel — APPROVED** [review]
So much goop.

File: static/img/logo.svg
Comment (bryanoltman) [reply static/img/logo.svg:1]: 🔵

**Review by bryanoltman — APPROVED** [review]

File: static/img/logo.svg
Comment (felangel) [reply static/img/logo.svg:1]: When we request a new logo we should get a vectorized version of it and ideally be able to support light/dark mode variants.

**Pull request merged**